addons/gamification/tests/test_karma_tracking.py

```diff
@@ -0,0 +1,211 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from datetime import datetime
+from odoo.tests import common
+
+
+class TestKarmaTrackingCommon(common.SingleTransactionCase):
+
+    def setUp(self):
+        super(TestKarmaTrackingCommon, self).setUp()
+        self.env['gamification.karma.tracking'].create([
```

Comment:
You could define a small tool method to create trackings between bounds like a range (min, max, steps) and call it in tests setup to reduce diff size.